CMakeLists.txt

```diff
@@ -88,6 +88,15 @@ target_include_directories(jwt-cpp SYSTEM INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/
 add_library(turtle INTERFACE)
 target_include_directories(turtle SYSTEM INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/3rd-party-libs/turtle/include)
 
+add_library(grpc INTERFACE)
```

Comment:
I am not sure, if it is good idea to integrate grpc like this. Is it able to build? Because I think grpc is not header only library

Reply:
You are right. I will change that. I thought that otherwise I could not use the grpcpp header files in greeter_client.cc and greeter_server.cc (e.g. <grpcpp/grpcpp.h>, <grpcpp/health_check_service_interface.h> etc.).